packages/components/text-field/src/text-field.stories.ts

```diff
@@ -189,62 +186,6 @@ export const Size: StoryObj = {
   }
 };
 
-export const Hint: StoryObj = {
-  render: () => {
-    return html`
-      <style>
-        form {
-          display: flex;
-          flex-direction: column;
-        }
-        sl-text-field {
-          margin-block-end: 1rem;
-        }
-      </style>
-      <form>
-        ${sizes.map((size, id) => {
-          return html`
-            <sl-label for="form-text-field-${id}">Nickname</sl-label>
-            <sl-text-field
-              .id=${`form-text-field-${id}`}
-              .size=${size}
-              hint-text="What would you like people to call you?"
-            ></sl-text-field>
-          `;
-        })}
-        <sl-label for="input4">Nickname</sl-label>
-        <sl-text-field
-          id="input4"
-          disabled
-          hint-text="What would you like people to call you?"
-          value="Disabled input"
-        ></sl-text-field>
-      </form>
-    `;
-  }
-};
-
-export const RichLabelHint: StoryObj = {
-  render: () => html`
-    <style>
-      form {
-        display: flex;
-        flex-direction: column;
-      }
-    </style>
-    <form>
-      <sl-label for="input">
-        <label slot="label">Custom <i>label</i></label>
-      </sl-label>
-      <sl-text-field id="input">
-        <div slot="hint-text">
-          Hint is an accessible way to provide <strong>additional information</strong> that might help the user
-        </div>
-      </sl-text-field>
-    </form>
-  `
-};
-
 export const ErrorMessageSizes: StoryObj = {
```

Comment:
Can you please update this story?
Error messages are not visible right now. :)

<img width="914" alt="image" src="https://github.com/sl-design-system/components/assets/111562742/0d9aeab4-b90c-4341-8ea5-7a373180cda4">